test/browser/render.test.js

```diff
@@ -173,7 +179,7 @@ describe('render()', () => {
 		expect(scratch.innerHTML).to.equal('');
 	});
 
-	it('should throw an error on function children', () => {
+	it.skip('should throw an error on function children', () => {
```

Comment:
function children don't seem to pass createElement making them victim to the _ check

Reply:
How does React handle this case?

Reply:
Well are functions as children allowed? Normally it should throw and now they won't get passed. Don't really know what we should do

Reply:
React produces this detailed error if you try to do this:

<img width="521" alt="Screenshot 2019-04-14 23 13 57" src="https://user-images.githubusercontent.com/2458/56099882-07b39780-5f0b-11e9-925a-3ced1d2c3947.png">

This logic could be moved to the `vnode` validation hook in `preact/debug` perhaps? The current behaviour on this branch of rendering nothing if a function child is passed to a DOM node seems reasonable to me, although that should have a test if it is agreed upon.

Reply:
Is this in scope of this pr?